app/javascript/components/issue_monograms.vue

```diff
@@ -67,17 +68,48 @@ export default {
     defaultThumbnailUrl: {
       type: String,
       default: null
+    },
+    formId: {
+      type: String,
+      default: 'nested_new_numismatics_monogram'
     }
   },
   data: function () {
     return {
       attachedMembers: this.members.filter((e) => e.attached)
     }
   },
+  computed: {
+    form: function () {
+      // The form should be a root/parent component
+      return document.getElementById(this.formId)
+    }
+  },
+  created: function () {
+    // If the form is a root/parent component, this would be $emit elsewhere
+    // this.form.addEventListener('attach-monogram', this.addAndAttachMember)
+    window.addEventListener('attach-monogram', this.handleAttachMember)
```

Comment:
I don't like using `window` here, I can try and target the `form` after retrieving it with `document.getElementById`